ingenialink/dictionary.py

```diff
@@ -213,7 +232,7 @@ class Dictionary(ABC):
     """Number of subnodes in the dictionary."""
     categories: DictionaryCategories
     """Instance of all the categories in the dictionary."""
-    errors: DictionaryErrors
+    errors: Dict[int, "DictionaryError"]
```

Comment:
I'm find with this simplification, but then the DictionaryErrors class serves no purpose, so the parsing method should be moved here